diff -crB vimrc/.vim/bundle/snipmate.vim/snippets/c.snippets snipmate.vim/snippets/c.snippets
*** vimrc/.vim/bundle/snipmate.vim/snippets/c.snippets	2012-08-20 15:27:52.440306000 +0200
--- snipmate.vim/snippets/c.snippets	2012-08-20 15:30:26.919069000 +0200
***************
*** 109,113 ****
  snippet fpr
  	fprintf(${1:stderr}, "${2:%s}\n"${3});${4}
  # This is kind of convenient
! snippet .
  	[${1}]${2}
--- 109,113 ----
  snippet fpr
  	fprintf(${1:stderr}, "${2:%s}\n"${3});${4}
  # This is kind of convenient
! snippet [
  	[${1}]${2}
diff -crB vimrc/.vim/bundle/snipmate.vim/snippets/python.snippets snipmate.vim/snippets/python.snippets
*** vimrc/.vim/bundle/snipmate.vim/snippets/python.snippets	2012-08-20 15:27:52.472305000 +0200
--- snipmate.vim/snippets/python.snippets	2012-08-20 15:30:26.921070000 +0200
***************
*** 47,53 ****
  # Lambda
  snippet ld
  	${1:var} = lambda ${2:vars} : ${3:action}
! snippet .
  	self.
  snippet try Try/Except
  	try:
--- 47,53 ----
  # Lambda
  snippet ld
  	${1:var} = lambda ${2:vars} : ${3:action}
! snippet '.
  	self.
  snippet try Try/Except
  	try:
